src/py-opentimelineio/opentimelineio/opentime.py

```diff
@@ -29,12 +48,17 @@ def to_timecode(rt, rate=None, drop_frame=None):
 
 
 def to_frames(rt, rate=None):
+    """Turn a :class:`~RationalTime` into a frame number."""
     return rt.to_frames() if rate is None else rt.to_frames(rate)
 
 
 def to_seconds(rt):
+    """Convert a :class:`~RationalTime` into float seconds"""
     return rt.to_seconds()
 
 
 def to_time_string(rt):
+    """Convert this timecode to time with microsecond, as formated in
```

Comment:
note that the documentation on that page is incorrect and misleading. We should not link it.

The correct information is here (thanks to @jchen9 for the reference) ~ https://github.com/FFmpeg/FFmpeg/blob/83e1a1de8833845224948e5d002355c03dd117d5/fftools/ffprobe.c#L412

```suggestion
    """Convert this timecode to time as used by ffmpeg, formatted as hh:mm:ss where ss is an integer or decimal number
```

Reply:
Would https://ffmpeg.org/ffprobe.html#Main-options be a better documentation?

```
See the [documentation of the -sexagesimal flag of ffprobe](https://ffmpeg.org/ffprobe.html#Main-options)
```
?
Or would you prefer that we simply don't link to anything and just describe the format OTIO accepts?

Reply:
The documentation at ffmpeg.org is incorrect there as well. It says seconds.milliseconds, but the code in ffprobe simply says %f with seconds as the argument. Their documentation explicitly says 3.5 means 3 and a half seconds, not 3 seconds and 5 milliseconds. I'm comfortable with stating this value, as interpretedby otio is compatible with ffprobe, but not comfortable at pointing to a misleading explanation. That's why I rewrote the explanations here as "ss means an integer or a decimal number" because that's what the ffprobe code does. 

As always, if I've misread the code, I'd appreciate correction, because I don't want to insist on something incorrect. I'm confused by the apparent mismatch of the documentation and the code, frankly. In general I haven't seen this kind of problem with ffmpeg before.

Reply:
So I simply went with your suggestion.